diffkemp/simpll/simpll_build.py

```diff
@@ -1,7 +1,18 @@
 #!/usr/bin/env python3
 from cffi import FFI
 from subprocess import check_output
-from diffkemp.utils import get_simpll_build_dir
+import os
+import sys
+
+def get_simpll_build_dir():
```

Comment:
This is copied from utils and I am not sure how to handle this properly, as I need access to the build directory during build time.

But we cannot import diffkemp, because at the time of using this script it is not yet accessible.

I will look into this again.